domain/src/test/java/org/oppia/android/domain/oppialogger/OppiaLoggerTest.kt

```diff
@@ -46,6 +68,122 @@ class OppiaLoggerTest {
   @Inject
   lateinit var oppiaLogger: OppiaLogger
 
+  private val TEST_VERBOSE_EXCEPTION = Throwable(TEST_VERBOSE_LOG_EXCEPTION)
+  private val TEST_DEBUG_EXCEPTION = Throwable(TEST_DEBUG_LOG_EXCEPTION)
+  private val TEST_INFO_EXCEPTION = Throwable(TEST_INFO_LOG_EXCEPTION)
+  private val TEST_WARN_EXCEPTION = Throwable(TEST_WARN_LOG_EXCEPTION)
+  private val TEST_ERROR_EXCEPTION = Throwable(TEST_ERROR_LOG_EXCEPTION)
+
+  @Test
+  fun testConsoleLogger_logVerboseMessage_checkLoggedMessageIsCorrect() {
+    ShadowLog.reset()
+    oppiaLogger.v(TEST_VERBOSE_LOG_TAG, TEST_VERBOSE_LOG_MSG)
+    val log = ShadowLog.getLogs().last()
+    assertThat(log.tag).isEqualTo(TEST_VERBOSE_LOG_TAG)
+    assertThat(log.msg).isEqualTo(TEST_VERBOSE_LOG_MSG)
+    assertThat(log.type).isEqualTo(Log.VERBOSE)
+  }
+
+  @Test
+  fun testConsoleLogger_logVerboseMessageWithException_checkLoggedMessageIsCorrect() {
+    ShadowLog.reset()
+    oppiaLogger.v(TEST_VERBOSE_LOG_TAG, TEST_VERBOSE_LOG_MSG, TEST_VERBOSE_EXCEPTION)
+    val log = ShadowLog.getLogs().last()
+    assertThat(log.tag).isEqualTo(TEST_VERBOSE_LOG_TAG)
+    assertThat(log.msg).isEqualTo(
+      "$TEST_VERBOSE_LOG_MSG\n${Log.getStackTraceString(TEST_VERBOSE_EXCEPTION)}"
+    )
```

Comment:
Here & elsewhere: I suggest splitting these into two separate assertThat() lines and using contains() instead of including the newline. It's a bit less fragile that way since this is assuming just one newline is always present between the items.

Reply:
done